docs/_docs/monitoring-metrics/system-views.adoc

```diff
@@ -168,6 +168,35 @@ This view exposes information about currently running compute tasks started by a
 |USER_VERSION | string |  Task user version
 |===
 
+== JOBS
+
+This view exposes information about currently running compute job started on a node.
+In the Ignite, Job defines part(step) of the compute task
+
+[{table_opts}]
+|===
+|NAME | TYPE |    DESCRIPTION
+|ID | UUID | Job ID
+|SESSION_ID | UUID | Session ID. Note, `SESSION_ID` equal to `TASKS.SESSION_ID` for the jobs that belongs to the specific task
+|ORIGIN_NODE_ID | UUID | Originating node id
```

Comment:
Is this the id of the 1) node that started the job or the 2) node where the job will be executed?

If #1 then say "The id of the node that started the job", otherwise "The id of the node where the job is being executed"

Reply:
Your first statement correct.
Fixed. Thanks.